### ISSUE TITLE
error: no member named 'Kind' in 'mlir::Value'

### ISSUE THREAD

**Issue by 3b3be5bc-be97-4c5a-a058-de8ddaf7c081** [llvm/llvm-project#48872]
|  |  |
| --- | --- |
| Bugzilla Link | [49528](https://llvm.org/bz49528) |
| Version | 11.0 |
| OS | Linux |

## Extended Description 
Xubuntu 20.10.

From the Ubuntu distribution
clang++ --version
Ubuntu clang version 11.0.0-2

git command executed close to 9pm Mar. 9, 2021, Mountain Standard Time.
git clone https://github.com/llvm/llvm-project.git
cd llvm-project
mkdir build
cd build

cmake -G Ninja -DLLVM_ENABLE_PROJECTS="clang;llvm;clang-tools-extra;compiler-rt;debuginfo-tests;flang;libc;libclc;libcxx;libcxxabi;libunwind;lld;lldb;mlir;openmp;parallel-libs;polly;pstl;utils" -DLLVM_USE_LINKER=lld -DCMAKE_BUILD_TYPE="Release" -DLLVM_TARGETS_TO_BUILD=X86 -DLLVM_ENABLE_LIBPFM=OFF -DRUN_HAVE_GNU_POSIX_REGEX=0 -DLLVM_LIBC_ENABLE_LINTING=ON -Wno-dev ../llvm &> ~/Documents/cmake.log

ninja -j 31 &> ~/Documents/ninja.log

From ninja.log
/home/nnelson/Documents/llvm-project/debuginfo-tests/llvm-prettyprinters/gdb/mlir-support.cpp:13:33: error: no member named 'Kind' in 'mlir::Value'
                   mlir::Value::Kind::TrailingOpResult});
                   ~~~~~~~~~~~~~^
/home/nnelson/Documents/llvm-project/debuginfo-tests/llvm-prettyprinters/gdb/mlir-support.cpp:27:23: error: no matching function for call to 'get'
auto FileLineColLoc = mlir::FileLineColLoc::get("file", 7, 8, &Context);
                      ^~~~~~~~~~~~~~~~~~~~~~~~~
tools/mlir/include/mlir/IR/BuiltinLocationAttributes.h.inc:50:27: note: candidate function not viable: no known conversion from 'const char [5]' to '::mlir::MLIRContext *' for 1st argument
    static FileLineColLoc get(::mlir::MLIRContext *context, StringRef filename, unsigned line, unsigned column);
                          ^
tools/mlir/include/mlir/IR/BuiltinLocationAttributes.h.inc:49:27: note: candidate function not viable: requires 3 arguments, but 4 were provided
    static FileLineColLoc get(Identifier filename, unsigned line, unsigned column);
                          ^
2 errors generated.

**Comment by gnaggnoyil** [comment]
Ping.
Is there any updates regarding this build failure?

**Comment by chsigg** [comment]
Sorry, forgot to close. Yes, this was fixed in 635f8f3.
